src/ralph/admin/filters.py

```diff
@@ -32,7 +34,33 @@ def date_format_to_human(value):
     return value
 
 
-class BaseCustomFilter(SimpleListFilter):
+class BaseCustomFilter(FieldListFilter):
+
+    """Base class for custom filters."""
+
+    def __init__(self, field, request, params, model, model_admin, field_path):
+        self.lookup_kwarg = field_path
+        self.parameter_name = field_path
```

Comment:
since we're inheriting from `FieldListFilter` now, parameter_name field is no longer necessary - you could use `self.field_path` instead